### PR TITLE
Optimize the application query, and add hide application feature

### PR DESCRIPTION
**Pull request by petterroea**

### CODE REVIEW

**Review by simsine — APPROVED** [review]

File: phoenixRest/models/crew/application.py [hunk 1]
```diff
@@ -73,7 +76,7 @@ def __json__(self, request):
             'uuid': str(self.uuid),
             'crews': self.crews,
             'event': self.event,
-            'user': map_user_with_secret_fields(self.user, request),
```
Comment (bernes1) [inline phoenixRest/models/crew/application.py:76]: why did you change this

**Review by bernes1 — APPROVED** [review]

**Review by HenrikEngen — APPROVED** [review]

File: phoenixRest/models/crew/application.py (hunk 1 above)
Comment (petterroea) [reply phoenixRest/models/crew/application.py:76]: In general I don't want `__json__` to return a lot of data. This is pretty expensive, and generates many queries.
